twitcher/__init__.py

```diff
@@ -35,21 +35,29 @@ def main(global_config, **settings):
     )
     authz_policy = ACLAuthorizationPolicy()
 
+
     config = Configurator(
         settings=settings,
         authentication_policy=authn_policy,
         authorization_policy=authz_policy
     )
-    from magpie.models import get_user
-    config.set_request_property(get_user, 'user', reify=True)
+
 
     # include twitcher components
     config.include('twitcher.config')
     config.include('twitcher.frontpage')
-    #config.include('twitcher.rpcinterface')
     config.include('twitcher.owsproxy')
     config.include('twitcher.wps')
 
+    auth_method = config.get_settings().get('twitcher.ows_security_provider', None)
+    if auth_method == 'magpie':
```

Comment:
Do previous 'config' operations require the `magpie_secret` value and auth policies right away?
If no, I would move all Magpie-specific operations under this if.
All Magpie imports could be under this if too, so that someone wanting to run Twitcher without any Magpie integration doesn't hit a module not found error or something similar.

Reply:
I intend to refactor the if auth_method == magpie later on before mergin back with birdhouse